src/main/webapp/static/css/styles.css

```diff
@@ -207,3 +205,7 @@ form {
 #formLoadingOverlay > .preloader-wrapper {
     top: 50%;
 }
+
+#github-button-spacer {
+    margin-right: 85px; /* The GitHub button is 80px wide */
+}
```

Comment:
This code is probably not used anywhere, and not relevant for the "Connect Wallet" button, right? So it can probably be removed.

```suggestion
```

Reply:
Done!